tests/telemetry/test_telemetry.py

```diff
@@ -182,8 +186,44 @@ def test_virtualdb_table_streaming(duthosts, enum_rand_one_per_hwsku_hostname, p
                  "Timestamp markers for each update message in:\n{0}".format(result))
 
 
-def invoke_py_cli_from_ptf(ptfhost, cmd):
-    ptfhost.shell(cmd)
+def invoke_py_cli_from_ptf(ptfhost, cmd, results=[""]):
```

Comment:
>results [](http://example.com/codeflow?start=41&length=7)

do not use empty array as function parameters. ref: [https://www.codiga.io/blog/python-avoid-empty-array-parameter/](https://www.codiga.io/blog/python-avoid-empty-array-parameter/)

Reply:
Done